openscm/scmdataframebase.py

```diff
@@ -827,6 +827,107 @@ def r(*args, **kwargs):
         x._resample_cls = get_resampler(self)
         return x.resample(time=rule, **kwargs)
 
+    def quantile_over(self, cols, quantile):
```

Comment:
Could this also take an axis parameter? Would it ever be interesting to see what the nth quantile of a timeseries is?

Reply:
yep e.g. you might want max and min of all the timeseries at each point over say models (so you'd be grouping by scenario)